plugins-api/src/main/scala/za/co/absa/enceladus/plugins/api/control/ControlPluginParams.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.plugins.api.control
+
+import za.co.absa.atum.model.RunState
+
+case class ControlPluginParams(datasetName: String,
```

Comment:
Are you sure this is a part of plugin API?

Reply:
No, thanks, removed.

(It was an attempt to replace `Map[String, String]` with a specific object, but we have already agreed that this is not desired for the API)